### ISSUE TITLE
[GWT] gwt compile failed

### ISSUE THREAD

**Issue by vinou31** [libgdx/libgdx#4500]
#### Issue details
I cloned the master depository because I needed to check on what was the problem with the freetypeFontGenerator and gwt. But before doing anything I wanted to test my clone on gwt so I launch a gwtCompile  on gdx-tests-gwt but it failed.
I took the liberty to change the gwt version to 2.8.0 since I spotted that in the gdx-test-gwt build.gradle it was still referencing 2.6.0. But the bug remain

#### Version of LibGDX and/or relevant dependencies
Libgdx 1.9.6-Snapshot
Gwt 2.8.0

#### Stacktrace
:tests:gdx-tests-gwt:compileGwt FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tests:gdx-tests-gwt:compileGwt'.
> Process 'command 'C:\Program Files\Java\jdk1.8.0_73\bin\java.exe'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [X] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by Tom-Ski** [comment]
Use superdev to run the gwt tasks.

**Comment by MobiDevelop** [comment]
Also, freetype is simply not going to work with GWT so I wouldn't spend much time on investigating why it doesn't work.

**Comment by vinou31** [comment]
ok well thanks for the hint about freetype.
Anyway, the superdev don't even work. I think it can be a missing module inheritance regarding the errors I get ?

Compiling module com.badlogic.gdx.tests.gwt.GdxTestsGwt
   Finding entry point classes
      [ERROR] Errors in 'file:/D:/Sources/libgdx/gdx/src/com/badlogic/gdx/Net.java'
         [ERROR] Line 80: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/GwtNet.java'
         [ERROR] Line 70: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/preloader/Preloader.java'
         [ERROR] Line 200: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
         [ERROR] Line 203: No source code is available for type java.io.ByteArrayInputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/com/badlogic/gdx/files/FileHandle.java'
         [ERROR] Line 78: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/gdx/src/com/badlogic/gdx/net/Socket.java'
         [ERROR] Line 41: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/preloader/Blob.java'
         [ERROR] Line 38: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/java/io/BufferedInputStream.java'
         [ERROR] Line 19: No source code is available for type java.io.FilterInputStream; did you forget to inherit a required module?
         [ERROR] Line 20: No source code is available for type java.io.InputStream; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/java/io/Reader.java'
         [ERROR] Line 33: No source code is available for type java.io.Closeable; did you forget to inherit a required module?
      [ERROR] Errors in 'file:/D:/Sources/libgdx/backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/emu/java/io/Writer.java'
         [ERROR] Line 30: No source code is available for type java.io.Closeable; did you forget to inherit a required module?
         [ERROR] Line 30: No source code is available for type java.io.Flushable; did you forget to inherit a required module?
      [ERROR] Unable to find type 'com.badlogic.gdx.tests.gwt.client.GwtTestStarter'
         [ERROR] Hint: Previous compiler errors may have made this type unavailable
         [ERROR] Hint: Check the inheritance chain from your module; it may not be inheriting a required module or a module may not be adding its source path entries properly
:tests:gdx-tests-gwt:draftCompileGwt FAILED

FAILURE: Build failed with an exception.

**Comment by Tom-Ski** [comment]
Yup, few updates weren't pushed. Should be good now, thanks for the issue report.

**Comment by vinou31** [comment]
thanks a lot indeed that change allow me to run the superdev mode.
Nevertheless, in order to compile with gwtcompile  and superdev mode on, I had to remove the current line from gdx-tests-gwt build.gradle : "enableClosureCompiler = true;"
I saw on the internet that closer compiler is no longer supported with gwt 2.8.0.

Also, I removed from the tests the following class : NonContinuousRenderingTest.java
It had errors with : 
`button1.addListener(new ChangeListener(){
			public void changed (ChangeEvent event, Actor actor) {
				new Thread(new Runnable(){
					public void run () {
						try {
							Thread.sleep(2000);
						} catch (InterruptedException ignored){}
						nextColor();
						Gdx.app.postRunnable(new Runnable(){
							public void run () {
								Gdx.app.log(str1, "Posted runnable to Gdx.app");
							}
						});
					}}).start();
				
			}});`

and other that look like this thread call. I hope it will help you find out that bugs this test.
